package-parser/package_parser/commands/generate_annotations/generate_annotations.py

```diff
@@ -121,6 +126,46 @@ def __get_unused_annotations(
             annotations.unused.append(UnusedAnnotation(formatted_name))
 
 
+def __get_enum_annotations(
+    usages: UsageStore, api: API, annotations: AnnotationStore
+) -> None:
+    """
+    Returns all parameters that are never used.
+    :param usages: UsageStore object
+    :param api: API object for usages
+    :param annotations: AnnotationStore object
+    :return: None
+    """
+    for methode, function in api.functions.items():
+        for parameter in function.parameters:
+            refined_type = parameter.refined_type.as_dict()
+            if "kind" in refined_type and refined_type["kind"] == "EnumType":
+                target = __qname_to_target_name(api, methode + "." + parameter.name)
+                enum_name = parameter.name
+                enum_name = re.sub("[^a-zA-Z_]", "", enum_name)
+                name_split = re.split("_", enum_name)
+                enum_name = ""
+                for name in name_split:
+                    enum_name += name.capitalize()
+                values = list(refined_type["values"])
+                values.sort()
+                pairs = []
+                for instance_name in values:
+                    string_value = instance_name
+                    instance_name = re.sub("[^a-zA-Z_]", "", instance_name)
+                    value_split = re.split("_", instance_name)
+                    instance_name = ""
+                    for split in value_split:
+                        instance_name += split.capitalize()
```

Comment:
Die selbe Funktionalität ist oben in Z. 144 bis 149 schon vorhanden. Bitte für Übersichtlichkeit beides in eine einzige Funktion auslagern.